core/src/helpers.ts

```diff
@@ -168,6 +172,80 @@ const buildRestList = <Model>(
   });
 };
 
+type TCreateSpecializedTransformersParams<TModel> = {
+  type: 'rest' | 'graphql';
+  buildFields?: (keyof TModel)[];
+};
+const createSpecializedTransformers = <TModel>({
+  type,
+  buildFields,
+}: TCreateSpecializedTransformersParams<TModel>) => {
+  return {
+    [type]: Transformer<TModel, TModel>(type, {
+      buildFields: buildFields,
+    }),
+  };
+};
+
+type TCreateSpecializedBuilderParams<TModel> = {
+  modelFieldsConfig: TModelFieldsConfig<TModel>;
+  type: 'rest' | 'graphql';
+  name: string;
+};
+const createSpecializedBuilder = <TModel>(
```

Comment:
Helper to create the new rest and graphql Builders we will be using with the new pattern.